js/apps/account-ui/src/api/methods.ts

```diff
@@ -109,8 +109,22 @@ export async function getCredentials({ signal, context }: CallOptions) {
   return parseResponse<CredentialContainer[]>(response);
 }
 
-export async function getLinkedAccounts({ signal, context }: CallOptions) {
-  const response = await request("/linked-accounts", context, { signal });
+export type LinkedAccountQueryParams = PaginationParams & {
+  search?: string;
+  linked?: boolean;
+};
+
+export async function getLinkedAccounts(
+  { signal, context }: CallOptions,
+  query: LinkedAccountQueryParams,
+) {
+  const response = await request("/linked-accounts", context, {
+    searchParams: Object.entries(query).reduce(
```

Comment:
Instead of `.reduce()` you could also do:

```ts
Object.fromEntries(Object.entries(query).map(([key, value]) => ([key, value.toString()])))
````

I kinda expected that `request()` would do this to `searchParams` automatically.

Reply:
atm this is the only request with parameters

Reply:
Yeah, this is an improvement that could be done at a later point